src/main/java/org/intellij/sdk/codesync/clients/CodeSyncClient.java

```diff
@@ -73,16 +72,10 @@ public Pair<Boolean, User> getUser(String accessToken) throws RequestError {
             return new Pair<>(false, null);
         }
 
-        try { JSONObject userPlanObject = (JSONObject) response.get("plan");
-            UserPlan userPlan = new UserPlan(
-                (Long) userPlanObject.get("SIZE"),
-                (Long) userPlanObject.get("FILE_COUNT"),
-                (Long) userPlanObject.get("REPO_COUNT")
-            );
+        try {
             User user = new User(
                 (String) response.get("email"),
-                (Long) response.get("repo_count"),
-                userPlan
+                (Long) response.get("repo_count")
```

Comment:
We don't need repo_count as well, kindly remove its usage too @syedgulahmed, thanks

Reply:
@Basit-qc Should we replace User class with string email? User class will have only one field named email.